### PR TITLE
[Excel] (Custom Functions) Update custom-functions-best-practices.md

### PR DESCRIPTION
**Pull request by keyur32**
Add note in the our troubleshooting docs

### CODE REVIEW

**Comment by o365devx** [comment]
OPS Build status updates of commit _[4bf5513](https://github.com/OfficeDev/office-js-docs-pr/pull/741/commits/4bf5513e8cd99ad0d0f59a01b04a0ed735fc514e)_: 

:clock10: **Preparing**: average preparing time is _44 sec(s)_

**Comment by o365devx** [comment]
OPS Build status updates of commit _[4bf5513](https://github.com/OfficeDev/office-js-docs-pr/pull/741/commits/4bf5513e8cd99ad0d0f59a01b04a0ed735fc514e)_: 

:clock10: **Incremental building**: average incremental building time is _29 sec(s)_

**Comment by o365devx** [comment]
OPS Build status updates of commit _[4bf5513](https://github.com/OfficeDev/office-js-docs-pr/pull/741/commits/4bf5513e8cd99ad0d0f59a01b04a0ed735fc514e)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/excel/custom-functions-best-practices.md](https://github.com/OfficeDev/office-js-docs-pr/blob/keyur32-patch-1/docs/excel/custom-functions-best-practices.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C2%5C8%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201902080224314200-741%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=hKoT0IEEc61u9fCpncRDGidaXgmUCFnz44zoRNX1jNY%3D&st=2019-02-08T02%3A20%3A51Z&se=2019-03-11T02%3A25%3A51Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/excel/custom-functions-best-practices.md [hunk 1]
```diff
@@ -33,9 +33,12 @@ function getComment(x) {
 
 ## Troubleshooting
 
-If you are testing your add-in in Office on Windows, you should enable **[runtime logging](../testing/troubleshoot-manifest.md#use-runtime-logging-to-debug-your-add-in)** to troubleshoot issues with your add-in's XML manifest file, as well as several installation and runtime conditions. Runtime logging writes `console.log` statements to a log file to help you uncover issues.
+1. If you are testing your add-in in Office on Windows, you should enable **[runtime logging](../testing/troubleshoot-manifest.md#use-runtime-logging-to-debug-your-add-in)** to troubleshoot issues with your add-in's XML manifest file, as well as several installation and runtime conditions. Runtime logging writes `console.log` statements to a log file to help you uncover issues.
+
+2. Your add-in will not be installed if one or more custom functions conflicts with a previously registered add-in's custom functions. In this case, you can either remove the existing add-in, or if you encounter this error while developing an add-in, you can specify a different namespace name in your manifest.
```
Comment (davidchesnut) [inline docs/excel/custom-functions-best-practices.md]: Is "install" the right term, or "load"?
Also is it possible to remove an add-in from a document? How do you do that?

Comment (keyur32) [reply docs/excel/custom-functions-best-practices.md]: load works better. I don't know do we have a document that explains how to uninstall add-ins? I couldn't find

**Comment by o365devx** [comment]
OPS Build status updates of commit _[5a1133c](https://github.com/OfficeDev/office-js-docs-pr/pull/741/commits/5a1133c751c7bb410df0885d01ab37f09da9bb89)_: 

:clock10: **Preparing**: average preparing time is _44 sec(s)_

**Comment by o365devx** [comment]
OPS Build status updates of commit _[5a1133c](https://github.com/OfficeDev/office-js-docs-pr/pull/741/commits/5a1133c751c7bb410df0885d01ab37f09da9bb89)_: 

:clock10: **Incremental building**: average incremental building time is _29 sec(s)_

**Comment by o365devx** [comment]
OPS Build status updates of commit _[5a1133c](https://github.com/OfficeDev/office-js-docs-pr/pull/741/commits/5a1133c751c7bb410df0885d01ab37f09da9bb89)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/excel/custom-functions-best-practices.md](https://github.com/OfficeDev/office-js-docs-pr/blob/keyur32-patch-1/docs/excel/custom-functions-best-practices.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C2%5C12%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201902122255414480-741%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=05czixM8%2FM4yR6HHsxjSSlBWNyslfiWGh5KyOWpiRB4%3D&st=2019-02-12T22%3A51%3A58Z&se=2019-03-15T22%3A56%3A58Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by mscharlock** [comment]
Hey @davidchesnut, I have a mail out to Linda about the correct terminology re: load/install/etc. If she doesn't have a consistent term, I'll send out a mail to the team. Since we've changed this to say load, I'm going to squash/merge.

**Pull request merged**